blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +224,18 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        if not row["connected"]:
+            connect_item = create_menuitem(_("_Connect"), get_icon("blueman", 16))
```

Comment:
How about a tooltip that lists the typical services it would cover, input an audio I know of but are there more that you know of?